srcpkgs/widelands/patches/musl.patch

```diff
@@ -10,15 +12,17 @@
 -  if (NOT CMAKE_SYSTEM_NAME MATCHES "kFreeBSD")
 -    find_library(EXECINFO_LIBRARY NAMES execinfo)
 -  endif (NOT CMAKE_SYSTEM_NAME MATCHES "kFreeBSD")
-+# Find libexecinfo for musl
-+find_library(EXECINFO_LIBRARY NAMES execinfo)
- 
+-
 -  # OpenBSD needs the X11 include directory in order to find GL/glu.h
 -  if (CMAKE_SYSTEM_NAME MATCHES "OpenBSD")
 -    find_package(X11 REQUIRED)
 -    include_directories(${X11_INCLUDE_DIR})
 -  endif ()
 -endif (CMAKE_SYSTEM_NAME MATCHES "FreeBSD" OR CMAKE_SYSTEM_NAME MATCHES "OpenBSD")
++# Find libexecinfo and libintl for musl
++find_library(EXECINFO_LIBRARY NAMES execinfo)
++find_library(INTL_LIBRARY NAMES intl)
```

Comment:
`find_package(Intl, REQUIRED)` would be better, I think.
I didn't dig into it, though.

Reply:
Anyway, does:
```
LDFLAGS="-lexecinfo -lintl"
```
works? If yes, it would be easier for us than patching

Reply:
I tried first with only `-lintl` and CMake completely ignored it. That's why I changed the patch :/

Reply:
CMake doesn't ignore it.
`cmake` just makes it hard to use correctly.
They pass `$LDFLAGS` before `*.a`, by order of filename, it couldn't file symbols in those libs.

`cmake` is `cmake`.